### PR TITLE
chore: remove envoy build from dashmate and release

### PR DESCRIPTION
**Pull request by strophy**
## Issue being fixed or feature implemented

Envoy was being built unnecessarily in CI on each run

## What was done?

Remove all Envoy-related build instructions. Envoy is now built and released separately from https://github.com/dashpay/docker-envoy/

## How Has This Been Tested?

In CI

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: packages/dashmate/configs/system/base.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ const { version } = require('../../package.json');
 
 const prereleaseTag = semver.prerelease(version) === null ? '' : `-${semver.prerelease(version)[0]}`;
 const dockerImageVersion = `${semver.major(version)}.${semver.minor(version)}${prereleaseTag}`;
+const envoyImageVersion = `${semver.major(version)}`;
```
Comment (shumkov) [inline packages/dashmate/configs/system/base.js:36]: Shouldn't we just use envoy base image version?

File: packages/dashmate/configs/system/base.js [hunk 2]
```diff
@@ -104,7 +105,7 @@ module.exports = {
     dapi: {
       envoy: {
         docker: {
-          image: `dashpay/envoy:${dockerImageVersion}`,
+          image: `dashpay/envoy:${envoyImageVersion}`,
           build: {
```
Comment (shumkov) [inline packages/dashmate/configs/system/base.js:109]: We should remove build section as well

File: packages/dashmate/configs/system/base.js (hunk 1 above)
Comment (strophy) [reply packages/dashmate/configs/system/base.js]: Sure, I've done a release to match the latest 1.22 branch from upstream, and pinned base config to `dashpay/envoy:1.22.11`

**Review by shumkov — DISMISSED** [review]
👍

**Review by shumkov — APPROVED** [review]

**Pull request merged**